llama_index/query_engine/retriever_query_engine.py

```diff
@@ -67,6 +68,7 @@ def from_args(
         output_cls: Optional[BaseModel] = None,
         use_async: bool = False,
         streaming: bool = False,
+        output_parser: Optional[BaseOutputParser] = None,
```

Comment:
I would prefer if this got passed into all response synthesizers, not just compact and refine 🙏🏻